.github/workflows/ci.yml

```
@@ -98,3 +98,22 @@ jobs:
 
     - name: Run effekt binary
       run: effekt.sh --help
+
+  build-and-test-kiama:
+    runs-on: ubuntu-latest
```

Nitpicks:
1. Do we even want a separate CI pass for this? (Compiling Effekt forces compiling Kiama anyway, no?)
2. If so, could we have a human-friendly name for the CI pass (so that it's nicer when reading it), something like:
```suggestion
  build-and-test-kiama:
    name: Build and test Kiama
    runs-on: ubuntu-latest
```